WebSecurityScanner/src/V1beta/Gapic/WebSecurityScannerGapicClient.php

```diff
@@ -929,248 +855,177 @@ public function stopScanRun($name, array $optionalArgs = [])
-     *                             format
-     *                             'projects/{projectId}/scanConfigs/{scanConfigId}/scanRuns/{scanRunId}'.
-     * @param array  $optionalArgs {
-     *                             Optional.
+     * @param ScanConfig $scanConfig   Required. The ScanConfig to be updated. The name field must be set to identify the
+     *                                 resource to be updated. The values of fields not covered by the mask
+     *                                 will be ignored.
+     * @param FieldMask  $updateMask   Required. The update mask applies to the resource. For the `FieldMask` definition,
+     *                                 see
+     *                                 https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#fieldmask
+     * @param array      $optionalArgs {
+     *     Optional.
      *
      *     @type RetrySettings|array $retrySettings
-     *          Retry settings to use for this call. Can be a
-     *          {@see Google\ApiCore\RetrySettings} object, or an associative array
-     *          of retry settings parameters. See the documentation on
-     *          {@see Google\ApiCore\RetrySettings} for example usage.
+     *           Retry settings to use for this call. Can be a
+     *           {@see Google\ApiCore\RetrySettings} object, or an associative array of retry
+     *           settings parameters. See the documentation on
+     *           {@see Google\ApiCore\RetrySettings} for example usage.
      * }
      *
-     * @return \Google\Cloud\WebSecurityScanner\V1beta\ListFindingTypeStatsResponse
+     * @return \Google\Cloud\WebSecurityScanner\V1beta\ScanConfig
      *
      * @throws ApiException if the remote call fails
-     * @experimental
      */
-    public function listFindingTypeStats($parent, array $optionalArgs = [])
+    public function updateScanConfig($scanConfig, $updateMask, array $optionalArgs = [])
     {
-        $request = new ListFindingTypeStatsRequest();
-        $request->setParent($parent);
-
+        $request = new UpdateScanConfigRequest();
+        $request->setScanConfig($scanConfig);
+        $request->setUpdateMask($updateMask);
         $requestParams = new RequestParamsHeaderDescriptor([
-          'parent' => $request->getParent(),
+            'scan_config.name' => $request->getScanConfig()->getName(),
         ]);
-        $optionalArgs['headers'] = isset($optionalArgs['headers'])
-            ? array_merge($requestParams->getHeader(), $optionalArgs['headers'])
-            : $requestParams->getHeader();
-
-        return $this->startCall(
-            'ListFindingTypeStats',
-            ListFindingTypeStatsResponse::class,
-            $optionalArgs,
-            $request
-        )->wait();
+        $optionalArgs['headers'] = isset($optionalArgs['headers']) ? array_merge($requestParams->getHeader(), $optionalArgs['headers']) : $requestParams->getHeader();
```

Comment:
The old style of this is more readable. As it's static, it should be easy to keep.

Reply:
I'd also like to see that with the next line as well.

Reply:
This was automatically formatted by the linter, and the generator codebase isn't designed for inserting newlines (AFAICT).